carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/factory/impl/DesktopFactory.java

```diff
@@ -171,4 +171,29 @@ private String getBrowserVersion(WebDriver driver) {
         }
         return browser_version;
     }
+    
+    /**
+	 * Sets browser window according to capabilites.resolution value, otherwise
+	 * maximizes window.
+	 * 
+	 * @param driver       - instance of desktop @WebDriver
+	 * @param capabilities - driver capabilities
+	 */
+	private void resizeBrowserWindow(WebDriver driver, DesiredCapabilities capabilities) {
+		try {
+			if (capabilities.getCapability("resolution") != null) {
```

Comment:
@hursevich, did you test it on mobile browsers as well?
I'm curious also about iOS (simulator) + Safari